app/kubernetes-ingress-controller/2.0.x/guides/getting-started-istio.md

```diff
@@ -70,735 +67,443 @@ Several examples using `kubectl` in this guide assume the test
+    clusterrolebinding.rbac.authorization.k8s.io/kiali created
+    role.rbac.authorization.k8s.io/kiali-controlplane created
+    rolebinding.rbac.authorization.k8s.io/kiali-controlplane created
+    service/kiali created
+    deployment.apps/kiali created
+    ```
 
-{% endnavtab %}
-{% navtab Response %}
+1.  Generate traffic for the BookInfo application, to create traffic metrics to view in Kiali:
 
-```console
-serviceaccount/kiali created
-configmap/kiali created
-clusterrole.rbac.authorization.k8s.io/kiali-viewer created
-clusterrole.rbac.authorization.k8s.io/kiali created
-clusterrolebinding.rbac.authorization.k8s.io/kiali created
-role.rbac.authorization.k8s.io/kiali-controlplane created
-rolebinding.rbac.authorization.k8s.io/kiali-controlplane created
-service/kiali created
-deployment.apps/kiali created
-```
+    ```console
+    $ COUNT=25 ; until [ $COUNT -le 0 ]; do curl -s -o /dev/null http://$PROXY_IP ; ((COUNT--)); done
+    ```
 
-{% endnavtab %}
-{% endnavtabs %}
+1.  In a production environment, you'd access the Kiali dashboard through the Kong 
+ingress. But this sample version of Kiali is meant for exploring only internal traffic on the 
+cluster. You can instead use the [port-forwarding][k8s-port-forwarding] functionality that `istioctl` 
+provides.
 
-Now we'll need to generate some traffic for our `BookInfo`
-app to populate metrics:
+    In a new terminal, run: 
 
-```console
-$ COUNT=25 ; until [ $COUNT -le 0 ]; do curl -s -o /dev/null http://$PROXY_IP ; ((COUNT--)); done
-```
-
-Since this sample version of Kiali isn't tuned for security and is meant to be
-cluster internal only, we won't expose this via Kong. Instead, we'll use the
-[port-forwarding][k8s-port-forwarding] functionality available in `istioctl` to
-tunnel our connection to the Kiali service.
+    ```console
+    $ istioctl dashboard kiali
+    ```
 
-Run the following command in a new terminal. This will run a `port-forward` to
-Kiali for you in the background and open it up in your web browser:
-
-{% navtabs %}
-{% navtab Command %}
-
-```console
-$ istioctl dashboard kiali
-```
-
-{% endnavtab %}
-{% navtab Response %}
-
-```console
-http://localhost:20001/kiali
-```
+    This runs a `port-forward` to Kiali in the background and opens it in your web browser. 
+    The response should look like:
 
-{% endnavtab %}
-{% endnavtabs %}
+    ```console
+    http://localhost:20001/kiali
+    ```
 
-This command should have automatically opened http://localhost:20001/kiali
-in your web browser, but if not then navigate to that page manually.
+    If http://localhost:20001/kiali doesn't open automatically 
+in your browser, navigate manually to the address.
 
 You're now connected to Kiali and have a window into the traffic moving across
 your mesh network. Familiarize yourself with Kiali and graphically view the
 topology for your `BookInfo` application's web requests:
 
 - Choose _Workloads_ from the menu on the left.
```

Comment:
is this a sequential list or just a bunch of unrelated things you can do? I know it's formatted as non-sequential, but some of the lines almost seem like they're related to each other.